cpu/efm32/include/periph_cpu.h

```diff
@@ -42,6 +43,13 @@ extern "C" {
 #endif
 
 #if (defined(ADC_COUNT) && (ADC_COUNT > 0)) || defined(DOXYGEN)
+/**
+ * @name    Power management configuration
+ * @{
+ */
+#define PROVIDES_PM_OFF     /**< pm_off function is provided */
+/** @} */
+
```

Comment:
This change was added with commit 9fc917580e8683a504ee517833144771b3940770 to fix a compilation issue in CI:
`cpu/cortexm_common/include/cpu.h` had to be included in `cpu/efm32/periph_cpu.h` so that 
`PROVIDES_PM_SET_LOWEST` is defined if only `periph_cpu.h` is included as header anywhere. Otherwise `pm_set_lowest` was defined multiple times if the `pm_layered` module is not used. Additionally, `PROVIDES_PM_OFF` had to be defined. I don't remember what application produced the compilation error but we can try it again without this change.

Reply:
I reverted commit 9fc917580e8683a504ee517833144771b3940770, lets see what Mudock says.

Reply:
It's failing ... I'd say it can be fixed by changing the `#define PROVIDES_PM_LAYERED_OFF` to `#define PROVIDES_PM_OFF` in this file. It should be the valid description how `periph_pm` is implemented ...

Reply:
Now I remember, the problem was the `riotbuild_dfu` test app. Probably, it is not using `pm_layered`.

Reply:
> I reverted commit [9fc9175](https://github.com/RIOT-OS/RIOT/commit/9fc917580e8683a504ee517833144771b3940770), lets see what Mudock says.

Hm, after the next round of CI compilation I had to add `#include "cpu.h"` in `cpu/efm32/include/periph_cpu.h` again to have `PROVIDES_PM_SET_LOWEST` defined, otherwise `pm_set` is defined multiple times. Also, it seems that both `PROVIDES_PM_LAYERED_OFF` and `PROVIDES_PM_OFF` must be defined in `cpu/efm32/include/periph_cpu.h`. `bootloaders/riotboot_dfu` does not use `pm_layered` and requires `PROVIDES_PM_OFF`.